tests/layer1/test_port_error.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import pytest
+import random
+import time
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import skip_release
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,  # disable automatic loganalyzer
+    pytest.mark.topology('any')
+]
+
+SUPPORTED_PLATFORMS = ["arista_7060x6"]
+cmd_sfp_presence = "sudo sfpshow presence"
+
+
+class TestMACFault(object):
+    @pytest.fixture(autouse=True)
+    def is_supported_platform(self, duthost, tbinfo):
+        if 'ptp' not in tbinfo['topo']['name']:
```

Comment:
@vdahiya12 Can we use the below API here instead?
https://github.com/sonic-net/sonic-mgmt/blob/a4bc0d78a02e5c8a68643369fd2ccfe3e4104daf/tests/common/testbed.py#L257

This is needed so that the test case runs for all variations of ptp (ptp-256, ptp-512 etc).
Also, the `get_testbed_type` API will likely need to be modified to return `ptp` if the topology name is `ptp-256`.

Reply:
Probably no need to call the API 
since that information is aleady stored in the 
common/testbed.py:    def get_testbed_type(self, topo_name):
common/testbed.py:            tb["topo"]["type"] = self.get_testbed_type(topo)

All we need to see that ptp is available in top name 

No need to change the API, since it uses regex